sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeTest.java

```diff
@@ -671,6 +675,36 @@ public void showMandatoryDownloadReadyDialogTest() {
         verify(mReleaseDownloader, times(2)).resume();
     }
 
+    @Test
+    public void checkUpdateReleaseAfterInterruptDownloading() {
+
+        /* Prepare data. */
+        mockStatic(DistributeUtils.class);
+        mockStatic(DeviceInfoHelper.class);
+        when(mReleaseDetails.getVersion()).thenReturn(1);
+        when(mReleaseDetails.isMandatoryUpdate()).thenReturn(true);
+        when(DeviceInfoHelper.getVersionCode(any(PackageInfo.class))).thenReturn(0);
+        when(DistributeUtils.loadCachedReleaseDetails()).thenReturn(mReleaseDetails);
+
+        /* Start distribute. */
+        start();
+
+        /* Start activity. */
+        Distribute.getInstance().onActivityResumed(mActivity);
+        ArgumentCaptor<ServiceCallback> httpCallback = ArgumentCaptor.forClass(ServiceCallback.class);
+        verify(mHttpClient).callAsync(anyString(), anyString(), eq(Collections.<String, String>emptyMap()), any(HttpClient.CallTemplate.class), httpCallback.capture());
+
+        /* Complete call with no new release (this will return the default mock mReleaseDetails with version 0). */
+        httpCallback.getValue().onCallSucceeded(mock(HttpResponse.class));
+
+        /* Disable and enable distribute module. */
+        Distribute.setEnabled(false);
+        Distribute.setEnabled(true);
+
+        /* Verify download is not called again. */
```

Comment:
> is not called again

`times(2)` ? Update comment or check. BTW did this test fail without the fix?

Reply:
yes, it fail without the fix